packages/access-policy/scripts/JestTransformer.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * https://babeljs.io/docs/en/config-files#jest
+ *
+ * Setup jest in Monorepo. This is to create a custom jest transformer file that wraps babel-jest.
+ *
+ * @type {Transformer}
+ */
+module.exports = require('babel-jest').createTransformer({
+  rootMode: 'upward',
+});
```

Comment:
Let's try defining this under the monorepo root, and importing it where required! No need to do this in this PR though, let's submit another PR for that